asv_bench/benchmarks/strings.py

```diff
@@ -248,6 +248,24 @@ def time_rsplit(self, dtype, expand):
         self.s.str.rsplit("--", expand=expand)
 
 
+class Extract:
+
+    params = (["str", "string", "arrow_string"], [True, False])
```

Comment:
prob should make a base class to have these params

Reply:
and include setup there

Reply:
good idea.